templates/markdown/partials/enum_string.md.tpl

```diff
@@ -4,9 +4,9 @@
             <ul>
                 {{- range $value, $display := .ValidValues -}}
                     {{- if $display -}}
-                        <li><strong><code>{{ $value }}</code>:</strong> {{ $display.Name }}</li>
+                        <li><strong><code style="white-space: nowrap">{{ $value }}</code>:</strong> {{ $display.Name }}</li>
                     {{- else -}}
-                        <li><strong><code>{{ $value }}</code></strong></li>
+                        <li><strong><code style="white-space: nowrap">{{ $value }}</code></strong></li>
                     {{- end -}}
```

Comment:
I'm happy to implement this based on your code recommendation, but I'm not very familiar with the templating here and was only tackling the HTML/CSS portions that I can test locally and feel confident about.

Reply:
Understood.  I'm not very familiar with this particular templating system, either, but I've worked with other templating systems which are similar, and figured it was easy enough for you to just try it.  😇

The challenging part here is the colon and whether we need/want it to be in "strong" face font; otherwise, it's pretty straightforward.

BTW, thanks for tackling the CSS stuff...I've managed to avoid learning much of that, so far.  😉